PromisesSwift.podspec

```diff
@@ -16,6 +16,7 @@ Pod::Spec.new do |s|
   s.osx.deployment_target  = '10.10'
   s.tvos.deployment_target = '9.0'
   s.watchos.deployment_target = '2.0'
+  s.swift_version = '5.2'
```

Comment:
Since we also know that this works with 5.0, how about doing:

```
s.swift_versions = ['5.0', '5.2']
```

Similar to:
https://github.com/Alamofire/Alamofire/blob/master/Alamofire.podspec

Reply:
Done